File: lab/capstone-cell-service/src/main/java/com/example/capstonecellservice/vo/ResponseCell.java
package com.example.capstonecellservice.vo;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Data;

import java.util.Date;

@Data
@JsonInclude(JsonInclude.Include.NON_NULL)
public class ResponseCell {

    private String cellId;

    private Integer totalCell;
    private Integer liveCell;
    private Integer deadCell;
    private Double viability;

    private Date createAt;
    private String url;
    private String userid;

}

File: android/Cellification/app/src/main/java/com/kyonggi/cellification/data/model/cell/ResponseCell.kt
package com.kyonggi.cellification.data.model.cell

import com.google.gson.annotations.SerializedName
import java.util.*

// 로그인 안했을시 사용? 저장은 못하게..
data class ResponseCell(
    @SerializedName("cellId")
    val cellId: String,
    @SerializedName("totalCell")
    val totalCell: Int,
    @SerializedName("liveCell")
    val liveCell: Int,
    @SerializedName("deadCell")
    val deadCell: Int,
    @SerializedName("viability")
    val viability: Double,
    @SerializedName("url")
    val url: String,
    @SerializedName("userid")
    val userId: String
)
